readme.md

```diff
@@ -13,7 +13,7 @@ Protractor-sync builds on protractor and provides:
 
 Pre-reqs:
 
-* Protractor (or something like grunt-protractor-runner, which includes it)
+* [Protractor](http://www.protractortest.org/#/) (follow the setup instructions)
```

Comment:
Good point, @mindywhitsitt!  I'm going to sync up with @shermaneric to make sure I have the Bb specific stuff in